sdk/tests/eval-run-decision-summary.test.ts

```
@@ -752,6 +765,247 @@ describe("the human renderer", () => {
+  it("names an unrecognized verdict, source or undecided reason instead of blanking it", () => {
+    // Guarding the lookups stopped `function Object() { [native code] }` from
+    // reaching a human, but a bare `?? ""` traded one broken sentence for
+    // another: `Decision summary:  ()` and a naked `Why:` read as a rendering
+    // bug rather than as a member this SDK predates. A vocabulary gaining a
+    // member is the EXPECTED case — an agent reading `Why: judgeNewThing` can
+    // go look it up, where a blank sends it nowhere.
+    const base = byName("measured-failure-at-every-stage").expected;
+    const text = formatEvalRunDecisionSummary({
+      ...base,
+      verdict: "verdictFromTheFuture" as unknown as typeof base.verdict,
+      verdictSource: "sourceFromTheFuture" as unknown as typeof base.verdictSource,
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)